### ISSUE TITLE
[Accessibility]: `<Link />` contrast inside of the `<Alert variant="danger" />`

### ISSUE THREAD

**Issue by valerybugakov** [sourcegraph/sourcegraph#38066]
### Audit type

General/Other

### Problem description

1. Use `sg start web-standalone`
2. The contrast of the link inside of the global alert `This build uses data from the proxied API` is lower then expected.

### Expected behavior

The `<Link />` contrast inside of the `<Alert variant="danger" />` is sufficient.

### Additional details

Potential fix: we should either change the `<Alert variant="danger" />` background color or change the `--link-color` to fix the contrast issue.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @taylorsperry @jasongornall - the "team/frontend-platform" label was applied to this issue.
